automation/capitanodoc/markdown.ts

```diff
@@ -24,6 +24,15 @@ import type { Category, Document, OclifCommand } from './doc-types';
 
 function renderOclifCommand(command: Category['commands'][0]): string[] {
 	const result = [`## ${ent.encode(command.name || '')}`];
+	if (!_.isEmpty(command.aliases)) {
```

Comment:
Can we do it we something that is not lodash? Ideally we would avoid including more usage of lodash as much as possible